en/docs/learn/adding-a-workflow-engine.md

```diff
@@ -0,0 +1,47 @@
+# Adding a Workflow Engine
+
+This section demonstrates how to configure the WSO2 Identity Server (IS)
+to connect to the [Business Process Management (BPM) component of WSO2
+Enterprise
+Integrator](https://docs.wso2.com/display/EI650/Working+with+Business+Processes) version 6.5.0
+. This can be done by creating a BPM Workflow Engine and configuring it
+accordingly. You can have many engines configured in different profiles
+and use one of them when creating a workflow.
+
+!!! info 
+    An embedded Business Process Management engine is included in WSO2
+    Identity Server. However, if you need more flexibility and more
+    extensibility with this feature than what is currently provided, it is
+    advisable to [integrate the Business Process Management (BPM) component
+    of WSO2 Enterprise
+    Integrator](../../learn/configuring-the-bpm-profile-of-wso2-ei-as-a-workflow-engine)
+    with the Identity Server.
+
+1.  To create a BPM profile click on **Configure \> Workflow Engine
+    Profiles \> Add**. The window below will be displayed.
+
+2.  Fill in the details using the descriptions below to guide you and
+    click **Add**.  
+    ![creating-a-bpm-profile](../assets/img/using-wso2-identity-server/creating-a-bpm-profile.png) 
```

Comment:
Please follow the guidelines in https://docs.google.com/document/d/14dZtE-f0BF46t5woX9k5aOawN1aIJ4WKaIFRqe7VyP8/edit#heading=h.i7s3q1sgw4a